corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_properties.html

```diff
@@ -307,6 +307,17 @@ <h4 class="panel-title panel-title-nolink">{% trans "Search and Claim Options" %
                   </button>
               </div>
           </div>
+          <div class="form-group">
+              <label for="custom-related-case-property" class="{% css_label_class %} control-label">
+                  {% trans "Custom related case ID property to include" %}
```

Comment:
I'm very open to changing this label and adding a help link - just figured I'd throw something up here for now since it's easy to change later.

Reply:
"Case property with additional case id to add to results"? Agreed about a help bubble.